Kakaotalk/UIConponents/MainListView.swift

```diff
@@ -1,8 +1,8 @@
 //
 //  MainListView.swift
```

Comment:
`MainListView` is more about the list itself and is not straightforward.  `UserListRow` can be a better name.